Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/tests.model.ts

```diff
@@ -18,9 +20,11 @@ export class Test {
     public createdDateTime: Date;
     public questionOrder: TestOrder;
     public optionOrder: TestOrder;
+    public allowTestResume: AllowTestResume;
```

Comment:
by default class properties are public. No need to specify explicit public keyword